packages/camel-catalog/kaoto-camel-catalog-maven-plugin/src/main/java/io/kaoto/camelcatalog/CamelCatalogProcessor.java

```diff
@@ -165,4 +169,21 @@ public String getPatternCatalog() throws Exception {
         });
         return JsonMapper.serialize(answer);
     }
+
+    private void populatePatterBlacklist() {
+        this.patternBlacklist.add("doCatch");
+        this.patternBlacklist.add("doFinally");
```

Comment:
Maybe these 2 need to be commented out for the time being.

![image](https://github.com/KaotoIO/kaoto-next/assets/16512618/847c71b5-9bee-4f44-b807-5c5f56fbe7e9)